mne/tests/test_epochs.py

```diff
@@ -1435,6 +1435,8 @@ def test_drop_epochs_mult():
 
 def test_contains():
     """Test membership API"""
+    # TODO This raw data does not contain seeg. The test cannot completely test
+    # membership.
```

Comment:
It should only take a few lines to add some fake channels, see here for an example of adding ECG to a raw instance that doesn't have it already:

https://github.com/mne-tools/mne-python/blob/master/mne/simulation/tests/test_raw.py#L68